configs/convmixer/convmixer-1024-20_10xb64_in1k.py

```diff
@@ -0,0 +1,12 @@
+_base_ = [
+    '../_base_/models/convmixer/convmixer-1024-20.py',
+    '../_base_/datasets/imagenet_bs64_convmixer_224.py',
+    '../_base_/schedules/imagenet_bs1024_adamw_swin.py',
+    '../_base_/default_runtime.py',
+]
+
+data = dict(samples_per_gpu=64)
```

Comment:
The `samples_per_gpu` in the dataset base settings is also 64, why add this line here?